Cdnfsd_TransparentCDN_Popup.php

```diff
@@ -0,0 +1,109 @@
+<?php
+namespace W3TC;
+
+
+
+class Cdnfsd_StackPath2_Popup {
+	static public function w3tc_ajax() {
+		$o = new Cdnfsd_TransparentCDN_Popup();
+
+		add_action( 'w3tc_ajax_cdn_transparentcdn_fsd_intro',
+			array( $o, 'w3tc_ajax_cdn_transparentcdn_fsd_intro' ) );
+		add_action( 'w3tc_ajax_cdn_stackpath2_fsd_configure_site',
```

Comment:
This file has many references to "stackpath2". Please update all files in this pull request to reference "transparent", to avoid conflicts.